apps/web/src/app/globals.css

```diff
@@ -5,7 +5,7 @@
 @layer base {
   body {
     @apply bg-brand-80 text-brand-20;
-    @apply font-secondary text-body-1;
+    @apply font-secondary font-secondary text-3xl leading-8;
```

Comment:
bro not sure why this duplicate `font-secondary` keeps appearing haha

Reply:
yeah I'm not sure either maybe it's something to do with my keyboard or a git rebase yeah I'm not sure either maybe it's something to do with my keyboard or a git rebase